### DISCUSSION TITLE
Can you please add support for fdf files?

### DISCUSSION THREAD

**Discussion by mat926** [sumatrapdfreader/sumatrapdf#3344] [Ideas]
fdf , not pdf.

**Comment by GitHubRulesOK** [comment]
I am unsure if MuPDF-accepts FDF (it should have the core ability to add those advanced AcroForm annotation inputs) However that's a complex process akin to editing and possibly even outside of MuTools abilities. By far the simplest is use one of the CLI editors like PDFtk or for a GUI there are many including alternatives such as https://github.com/Tadelsucht/BulkPDF
